app/admin/contributions.rb

```diff
@@ -1,23 +1,24 @@
 ActiveAdmin.register Contribution do
+  decorate_with ContributionDecorator
   permit_params :state, :link, :user_id, :company_id, :repository_id
   actions :index, :show, :new, :create
 
   menu parent: Contribution.model_name.human(count: 2), priority: 1
 
-  filter :company, as: :select, if: proc { current_user.super_admin? }
-  filter :user, as: :select, collection: proc { CompanyUsersCollectionQuery.new(current_user).call }
+  filter :company, as: :select, if: proc {current_user.super_admin?}
```

Comment:
Your editor is probably configured to remove spaces between `{}`, you should try to install editorconfig plugin on it.